mdoc/Makefile

```diff
@@ -852,6 +857,22 @@ check-ignored-namespace-generic:
 	$(MONO) $(PROGRAM) update Test/test-generic-ignored-namespace/ReadOnlySpan.dll -o Test/en.actual/
 
 	$(DIFF) Test/test-generic-ignored-namespace/en.expected-generic-ignored-namespace Test/en.actual/
+
+check-type-projection: Test/DocTest-TypeProjection.dll
+	rm -Rf Test/en.actual
+	rm -Rf Test/test-type-projection
+
+	mkdir Test/test-type-projection
+	mkdir Test/test-type-projection/DocTest-TypeProjection
+	mkdir Test/en.actual
+
+	cp Test/DocTest-TypeProjection.dll Test/test-type-projection/DocTest-TypeProjection
+	cp Test/TypeMap.xml Test/test-type-projection/
```

Comment:
You should give `TypeMap.xml` a different name in the `Test` directory ... something that makes it clearer which test it's related to. You can rename it after you copy with the `mv` command.